### PR TITLE
MCI Mapping

### PR DESCRIPTION
**Pull request by gigxz**

### CODE REVIEW

File: drivers/hmis_external_apis/app/jobs/hmis_external_apis/create_referral_job.rb [hunk 1]
```diff
@@ -48,7 +48,7 @@ def create_referral_postings(referral)
         end
       projects_ids_by_mper_id = external_id_map(
         cred: mper_cred,
-        scope: Hmis::Hud::Project,
+        scope: ::Hmis::Hud::Project,
```
Comment (gigxz) [inline drivers/hmis_external_apis/app/jobs/hmis_external_apis/create_referral_job.rb:51]: I needed all these to get tests to pass locally

File: drivers/hmis_external_apis/app/models/hmis_external_apis/oauth_client_result.rb [hunk 2]
```diff
@@ -1,3 +1,9 @@
+###
+# Copyright 2016 - 2022 Green River Data Analysis, LLC
```
Comment (eanders) [inline drivers/hmis_external_apis/app/models/hmis_external_apis/oauth_client_result.rb]:
```suggestion
# Copyright 2016 - 2023 Green River Data Analysis, LLC
```

File: drivers/hmis_external_apis/app/models/hmis_external_apis/mci_mapping.rb [hunk 3]
```diff
@@ -0,0 +1,103 @@
+###
+# Copyright 2016 - 2022 Green River Data Analysis, LLC
```
Comment (eanders) [inline drivers/hmis_external_apis/app/models/hmis_external_apis/mci_mapping.rb]: Pretty sure the rest of the app has been updated to 2023, not sure why these were missed.

**Review by eanders — APPROVED** [review]
I think this looks good.  A couple of weird things with the copyright noted inline.

**Comment by gigxz** [comment]
Good catch, fixed!

**Pull request merged**